src/css/general.css

```diff
@@ -291,7 +291,26 @@ th {
 }
 
 /* Formatting for KaTeX math blocks */
-div.equation {
+.equation-block {
```

Comment:
Can you clarify for me that this shouldn't be katex-block from the vue-shpinx-xml package MathBlock?

Reply:
It should ... and it's been changed.  You're looking at an old commit?